source/dagger/gameplay/plight/plight_combat.cpp

```diff
@@ -62,7 +62,7 @@ void PlightCombatSystem::Run()
 
 			//Stamina will not be affecting running , it will be used for special movement like dashing or rolling when it gets implemented (used on running for example here)
 
-			if (character.running) {
+			/*if (character.running) {
```

Comment:
do u need running still?

Reply:
I left it commented in case we need it during development for some reason , or if we decide to add cost for running as 
well